include/veriblock/entities/merkle_path.hpp

```diff
@@ -46,6 +45,25 @@ struct MerklePath {
     ReadStream merkleStream(merkleBytes);
     return MerklePath::fromRaw(merkleStream, subject);
   }
+
+  static void toRaw(WriteStream& stream, const MerklePath& path) {
+    writeSingleFixedBEValue(stream, path.index);
+    writeSingleFixedBEValue(stream, (int32_t) path.layers.size());
+
+    const auto subjectSizeBytes = fixedArray((int32_t) path.subject.size());
+    writeSingleFixedBEValue(stream, (int32_t)subjectSizeBytes.size());
+    stream.write(subjectSizeBytes);
+
+    for (auto layer : path.layers) {
```

Comment:
do not make redundant copy
```suggestion
    for (const auto& layer : path.layers) {
```